orchagent/pfcactionhandler.h

```diff
@@ -13,6 +13,14 @@ extern "C" {
 using namespace std;
 using namespace swss;
 
+struct PfcWdHwStats
+{
+    uint64_t txPkt     = 0;
```

Comment:
>= 0 [](start = 23, length = 3)

Normally you don't need to provide a struct a ctor, or init values.

Reply:
In a C-style code, I'd normally write something like this
```
PfcWdHwStats stats = {};
```
To initialize all values.
Do you prefer this way over current implementation?

Reply:
Yes.

Reply:
Done